### DISCUSSION TITLE
Sortable grid dashboard

### DISCUSSION THREAD

**Discussion by timheerwagen** [clauderic/dnd-kit#880] [Q&A]
What is the best way to create a sortable grid dashboard where sortable items are not swapped automatically, but are moved down one level when a smaller item is moved above it. Also, I'm looking for a way to move items back and forth on free spaces within the container. Currently I can only swap items.

With SortableContext I can rearrange items, but some of them jump back and forth.
Would it make sense to do without the SortableContext and use a snapToGrid modifier instead?

![Unbenannt](https://user-images.githubusercontent.com/35360306/188005226-50cfe136-3d53-49f0-95a4-2c9d3a401528.png)

**Comment by taziksh** [comment]
This looks really cool! Would be great to see the Codesandbox if you're open to sharing

**Comment by timheerwagen** [comment]
@taziksh 
Unfortunately, I have not found a solution for Dnd-kit. So I had to use react-grid-layout for that. But if someone finds a solution I would be very pleased, because I would like to stay with Dnd-kit.
